docs/migration.md

```diff
@@ -0,0 +1,213 @@
+NAME                                                             READY   STATUS   RESTARTS   AGE
+migrate-cstor-pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d-52w98     1/1     Running    0          34s
+```
+```sh
+$ k logs -f migrate-cstor-pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d-52w98
+I0713 12:51:29.090705       1 cstor_volume.go:73] Migrating volume pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to csi spec
+I0713 12:51:29.122973       1 volume.go:807] Checking for a temporary policy of volume pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d
+I0713 12:51:29.126303       1 volume.go:817] Creating temporary policy pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d for migration
+I0713 12:51:29.264918       1 volume.go:179] Checking volume is not mounted on any application
+I0713 12:51:29.326575       1 volume.go:185] Retaining PV to migrate into csi volume
+I0713 12:51:29.332122       1 volume.go:528] Updating storageclass openebs-cstor-sparse with csi parameters
+I0713 12:51:29.630087       1 volume.go:311] Generating equivalent CSI PVC testclaim-busybox-0
+I0713 12:51:29.655155       1 volume.go:245] Recreating equivalent CSI PVC
+I0713 12:51:29.838898       1 volume_operations.go:133] Waiting for pvc testclaim-busybox-0 to go away
+I0713 12:51:34.879781       1 volume.go:350] Generating equivalent CSI PV pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d
+I0713 12:51:34.879807       1 volume.go:285] Recreating equivalent CSI PV
+I0713 12:51:34.889090       1 volume_operations.go:150] Waiting for pv pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to go away
+I0713 12:51:40.112757       1 volume.go:215] Creating CVC to bound the volume and trigger CSI driver
+I0713 12:51:40.207699       1 volume.go:892] Validating the migrated volume
+I0713 12:51:40.296914       1 volume.go:901] Waiting for cvc pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to become Bound, got: Pending
+I0713 12:51:50.309624       1 volume.go:951] Waiting for cv pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to come to Healthy state, got: 
+I0713 12:52:00.319407       1 volume.go:951] Waiting for cv pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to come to Healthy state, got: 
+I0713 12:53:11.175510       1 volume.go:951] Waiting for cv pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to come to Healthy state, got: Init
+I0713 12:53:21.177993       1 volume.go:951] Waiting for cv pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d to come to Healthy state, got: Init
+I0713 12:53:31.242181       1 volume.go:956] Patching the target svc with cvc owner ref
+I0713 12:53:31.336819       1 volume.go:1029] Cleaning up old volume resources
+I0713 12:53:31.714056       1 cstor_volume.go:80] Successfully migrated volume pvc-7ac10812-cc83-4fc5-a2e0-7d24f785e93d, scale up the application to verify the migration
+```
+**Note:** If target affinity was set to the old volume, the target pod will go into `pending` state after the migration is completed. Once the application is scaled up the target pod should automatically reschedule to the same node as application.
```

Comment:
We need to mention this [point](https://github.com/openebs/upgrade/pull/15#pullrequestreview-446437868) also.

Reply:
Done